packages/wallets/src/components/WalletListHeader/WalletListHeader.scss

```diff
@@ -0,0 +1,75 @@
+.wallets-list-header {
+    display: flex;
+    max-width: 123.2rem;
+    height: max-content;
+    width: 100%;
+    padding: 1.6rem 0 2.4rem;
+
+    @include mobile {
+        padding: 0.8rem 1.4rem;
+    }
+
+    &__label {
+        display: flex;
+        position: absolute;
```

Comment:
Not sure why do we need the `position: absolute` here. There's no top, left, right or bottom define stylings. You can remove it because it might lead to some UI problems.

Reply:
This is needed for the labels as it is positioned above the switcher. Removing it will place the label on top of the switcher:

<img width="130" alt="image" src="https://github.com/binary-com/deriv-app/assets/142860499/370aa415-53e8-47ef-bd9b-7e1bfcd5be50">

Reply:
Not defining the top, left, right and bottom values are intended to ensure the elements stack on top of each other with no relative translation

Reply:
Sorry I've missed that thanks for the explanation @lubega-deriv 👍🏼